### DISCUSSION TITLE
Specify relative path to preset in extend

### DISCUSSION THREAD

**Discussion by Ka0o0** [renovatebot/renovate#16983] [Request Help] [answered]
Hi,

I wondering if it's possible to specify relative paths to presets in the same repository. Given a preset `local>group/presets//initial-preset`. When this presets further extends a preset in the same repository, I'm always specifying the full path like `local>group/presets//other-preset`. Ideally, I would like to just specify `./other-preset` there and have Renovate resolve the relative path. This should also be true for the global config. 

Reading https://docs.renovatebot.com/config-presets/ didn't yield any information in this regard but maybe I've misunderstood something.

Thanks

**Comment by viceice** [comment, marked as answer]
not possible and we don't plan to change that, as it would make things more complex

**Reply by Ka0o0** [reply]
Okay thanks for the information.
